src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx

```diff
@@ -0,0 +1,321 @@
+
+  const userSafe = getUserSafe()
+
+  const showLoader = isLoading || !safeAppDetails
+
+  return (
+    <Container>
+      <StyledCard>
+        {showLoader ? (
+          <LoaderContainer>
+            <Loader size="md" />
+          </LoaderContainer>
+        ) : (
+          <>
+            {/* Safe App details */}
+            {safeAppDetails && (
+              <DetailsContainer>
+                <SafeIcon src={safeAppDetails.iconUrl} />
+                <DescriptionContainer>
+                  <SafeAppTitle size={'sm'}>{safeAppDetails.name}</SafeAppTitle>
+                  <div>{safeAppDetails?.description}</div>
+                </DescriptionContainer>
+              </DetailsContainer>
+            )}
+            <Separator />
+
+            {/* Available chains */}
+            {showAvailableChains && (
+              <>
+                <ChainLabel size="lg">Available networks</ChainLabel>
+                <ChainsContainer>
+                  {availableChains.map((chainId) => (
+                    <div key={chainId}>
+                      <NetworkLabel networkInfo={getChainById(chainId)} />
+                    </div>
+                  ))}
+                </ChainsContainer>
+                <Separator />
+              </>
+            )}
+
+            <ActionsContainer>
+              <UserSafeContainer>
+                <Title size={'xs'}>Use the dapp with your Safe!</Title>
+                {isWalletConnected ? (
+                  <>
+                    {userSafe ? (
+                      <div>not implemented</div>
+                    ) : (
+                      <>
+                        {/* Create new Safe */}
+                        <BodyImage>
+                          <Img alt="Vault" height={92} src={SuccessSvg} />
+                        </BodyImage>
+
+                        <Button
+                          data-testid={'create-new-safe-link'}
+                          size="lg"
+                          color="primary"
+                          variant="contained"
+                          component={Link}
+                          to={OPEN_SAFE_ROUTE}
+                        >
+                          <Text size="xl" color="white">
+                            Create new Safe
+                          </Text>
+                        </Button>
+                      </>
+                    )}
+                  </>
+                ) : (
+                  <>
+                    {/* Connect your Wallet */}
+                    <StyledProvider>
+                      <Provider
+                        info={<ProviderDisconnected />}
+                        open={open}
+                        toggle={toggle}
+                        render={(providerRef) =>
+                          providerRef.current && (
+                            <Popper
+                              anchorEl={providerRef.current}
+                              style={{ zIndex: 1301 }}
```

Comment:
Why do you need such a high `z-index`? It's a nit, but using such arbitrary values can cause issues with styling down the line.

Reply:
I am reusing the same component that we are using in our header.

![Captura de pantalla 2022-04-08 a las 13 17 05](https://user-images.githubusercontent.com/26763673/162426397-4792d46d-daa0-4eb3-b669-44c8a5839dd1.png)


The only style that I need to set manually here is the `z-index`, but I will use the `style(Popper)` instead of inline.

Reply:
Does it need be such a high number though?

Reply:
yes, because our header has a `z-index: 1301` value:

![Captura de pantalla 2022-04-08 a las 14 05 36](https://user-images.githubusercontent.com/26763673/162432224-1612e8eb-b506-4ccc-b6d4-6a05cccf5f4a.png)

The value is probably related with [the Material UI z-index values.](https://mui.com/customization/z-index/)